### DISCUSSION TITLE
How can I reach ComfyUI that runs at home from my office?

### DISCUSSION THREAD

**Discussion by daizhuo** [comfyanonymous/ComfyUI#609] [General]
I'm not a programmer, could you help me to do this. Thank you!
I want to reach ComfyUI that runs at home from my office.
I know how to do that in SD Webui, but don't know how to do that in ComfyUI.
How to add --listen, and can reach it from outside?
It should be easy as SD WebUi, right?
Thanks again.

**Comment by daizhuo** [comment]
127.0.0.1 is not my computer’s IP, I can not reach this address via DDNS  from outside.
My computer’IP should be 192.168.0.155.
I can reach 192.168.50.155:8188, not 127.0.0.1:8188

In WebUI, add - - listen, It’s IP would be 0.0.0.0:7860.  So I can reach it from outside.

Should ComfyUI do the same thing?

Thank you so much 😊

**Reply by comfyanonymous** [reply]
Yes you can use --listen in ComfyUI and it will listen on 0.0.0.0.

**Reply by daizhuo** [reply]
How to do that?
Sorry. I'm not a programmer.
I don't know where should I go for.
Thank you!

**Reply by TheBlueSky** [reply]
@daizhuo, you do that by starting ComfyUI with `python main.py --listen`

**Comment by Mikerhinos** [comment]
Add --listen (so _python PathToComfyUI\main.py --listen_ ) and don't forget to redirect port 8188 to your ComfyUI pc so your internet router knows where it must redirect the machine that will try to connect to _YourInternetIP:8188_
